include/fastdds/dds/domain/DomainParticipantFactory.hpp

```diff
@@ -94,6 +94,17 @@ class DomainParticipantFactory
             DomainParticipantListener* listener = nullptr,
             const StatusMask& mask = StatusMask::all());
 
+    /**
+     * Create a Participant.
+     *
+     * @param listener DomainParticipantListener Pointer (default: nullptr)
+     * @param mask StatusMask Reference (default: all)
+     * @return DomainParticipant pointer. (nullptr if not created.)
+     */
+    RTPS_DllAPI DomainParticipant* create_participant_with_default_profile(
+            DomainParticipantListener* listener = nullptr,
+            const StatusMask& mask = StatusMask::all());
```

Comment:
Since this is a new API, please follow best-practices, and do not use default values for arguments